input.css

```diff
@@ -151,9 +151,14 @@ pre {
   font-weight: bold;
 }
 
+.search-result-item:hover, .search-suggestion-item:hover {
+  background-color: #B3B3B3;
+}
 
-.search-result-item:hover {
-  background-color: #ecf6ff;
+.search-suggestion-item {
```

Comment:
I can adjust the styling for the search results to be more consistent with the changes added here for the suggestions in a separate story.